mls-rs/src/group/transcript_hash.rs

```diff
@@ -19,69 +21,36 @@ use crate::{
 
 use super::{AuthenticatedContent, ConfirmationTag};
 
-#[derive(Clone, PartialEq, Eq, MlsSize, MlsEncode, MlsDecode)]
-#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
-#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-pub struct ConfirmedTranscriptHash(
-    #[mls_codec(with = "mls_rs_codec::byte_vec")]
-    #[cfg_attr(feature = "serde", serde(with = "mls_rs_core::vec_serde"))]
-    Vec<u8>,
-);
-
-impl Debug for ConfirmedTranscriptHash {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        mls_rs_core::debug::pretty_bytes(&self.0)
-            .named("ConfirmedTranscriptHash")
-            .fmt(f)
+#[cfg_attr(not(mls_build_async), maybe_async::must_be_sync)]
+pub(crate) async fn create<P: CipherSuiteProvider>(
+    cipher_suite_provider: &P,
+    interim_transcript_hash: &InterimTranscriptHash,
+    content: &AuthenticatedContent,
+) -> Result<ConfirmedTranscriptHash, MlsError> {
+    #[derive(Debug, MlsSize, MlsEncode)]
+    struct ConfirmedTranscriptHashInput<'a> {
+        wire_format: WireFormat,
+        content: &'a FramedContent,
+        signature: &'a MessageSignature,
     }
-}
```

Comment:
One thing we should decide ... maybe core can have things like this in there as well. Doesn't need to change now, but it feels like it might be useful for these small crypto operations that will never change to exist in core to keep things all toegether

Reply:
Let's think about it. I'm not the greatest fan of the interim/confirmed transcript hash concept - one hash can be enough for security, and it could be computed more easily.